pkg/user/signer.go

```diff
@@ -327,11 +327,10 @@ func (s *Signer) getSignatureV2(sequence uint64, pubKey cryptotypes.PubKey, sign
 			SignMode:  signing.SignMode_SIGN_MODE_DIRECT,
 			Signature: signature,
 		},
+		PubKey:   pubKey,
```

Comment:
Shouldn't pubKey always be set on signature? Without this I was getting a panic (nil pointer) in x/auth/tx/adaptor.go (pubKey was nil on tx.AuthInfo)